html/entities.ts

```diff
@@ -12,6 +12,20 @@ const rawToEntityEntries = [
   ["'", "&#39;"],
 ] as const;
 
+const forbiddenCustomElementNames: string[] = [
```

Comment:
Few things:
1. Please uppercase, as this variable doesn't change.
2. `string[]` isn't needed.
3. Please link to WHATWG documentation for this list here.

Reply:
Actually, 2 and 3 are fine to leave.